server/command.go

```diff
@@ -262,6 +282,16 @@ func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 
 func (p *Plugin) runSettingCommand(args *model.CommandArgs, params []string, user *model.User) (string, error) {
 	if len(params) == 0 {
+		restrict, _, err := p.checkChannelPreference(args.ChannelId)
+		if err != nil {
+			p.API.LogError("Unable to check channel preference", "ChannelID", args.ChannelId, "Error", err.Error())
+			return "Error occurred while viewing zoom settings", nil
+		}
+
+		if restrict {
+			return "Updating zoom settings is disabled for the current channel.", nil
```

Comment:
```suggestion
			return "Updating zoom settings is disabled for this channel.", nil
```
Just to be consistant